namespaces/live.cloud-platform.service.justice.gov.uk/pfs-education-laptop-provisioner-dev/resources/serviceaccount.tf

```diff
@@ -0,0 +1,10 @@
+module "serviceaccount" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-serviceaccount?ref=0.7.4"
```

Comment:
https://github.com/ministryofjustice/cloud-platform-terraform-serviceaccount/releases/tag/0.7.5 please, we always want modules on latest

Reply:
i have updated to the code to show latest. I created using the cli and it came from the template. I expect this will need updating.